### ISSUE TITLE
Likers Blocker Button Doesn't Show up

### ISSUE THREAD

**Issue by SamParkerGOP** [dmstern/likers-blocker#203]
Likers blocker on twitter doesn't work and I'm thinking it's because Elon changed how the list of "likes" is displayed now. Same for quotes and r/t, etc.

Is there a workaround?

**Comment by aghanki** [comment]
Exactly, it doesn't show, but when you go to a list and want to view the members, it shows. Please fix it; I really need this.
![image](https://github.com/dmstern/likers-blocker/assets/50052329/e072e283-4d8a-4d1a-9897-3882cc38fa63)
